guide/payments/privacy.md

```diff
@@ -0,0 +1,137 @@
+
+</div>
+
+### Increase anonymity by collaborating with others
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/privacy/coin-join.jpg"
+   retina = "/assets/images/guide/payments/privacy/coin-join@2x.jpg"
+   alt-text = "Example CoinJoin transaction with 3 participants with 1 input each, which get broken up into 5 equal outputs."
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+The "common input ownership" heuristic assumes that in a transaction, it is most likely that all inputs belong to the same owner. [CoinJoins]({{ "/guide/glossary/#coinjoin" | relative_url }}) is a technique where multiple participants collaborate on a transaction to break this heuristic. It becomes more difficult (although not impossible) to determine who the individuals who own the outputs are since all the outputs typically have the same denomination. With CoinJoins, as with any other anonymity network, a large and diverse group of participants provides the best protection.
+
+Users still have to be mindful of how the UTXOs they received from the CoinJoin are spent. For instance, spending them together in a single transaction would unravel the anonymity gains from participating in the CoinJoin.
+
+</div>
+
+### Design with privacy in mind
+
+Thinking about privacy is critical to keep in mind during the design process. Your users will not have the same level of knowledge of how to use Bitcoin privately. It becomes especially critical to understand the implications of their privacy degrading actions at the point of creating a transaction or requesting a payment.
+
+While there is no perfect solution that will guarantee 100% privacy, try always to minimize how much information gets shared to the most essential. Also, consider ways to inform and prevent users of actions that negatively impact their privacy as they use your product.
+
+<!--
+There is no perfect solution to guarantee 100% privacy that lasts forever because things can be revealed over time. Since transactions are forever public, even if all precautions are taken at the time of payment to ensure the highest degree of anonymity, future behaviors of the wallet owner or transacting parties can still degrade previously attained privacy. A high amount of diligence is necessary whenever users are transacting with Bitcoin. The product should be able to guide, inform, and prevent them against privacy degrading actions.
+-->
+
+[^1]: [Bitcoin.org - Protecting your privacy](https://bitcoin.org/en/protect-your-privacy)
+[^2]: [Wasabi Wallet Docs - Transaction Surveillance Companies](https://docs.wasabiwallet.io/why-wasabi/TransactionSurveillanceCompanies.html#attempt-to-invade-privacy)
+[^3]: [Top Seven Ways Your Identity Can Be Linked to Your Bitcoin Address](https://99bitcoins.com/know-more-top-seven-ways-your-identity-can-be-linked-to-your-bitcoin-address/)
```

Comment:
Could add a link to further reading on the Bitcoin wiki, they have a great [page here](https://en.bitcoin.it/wiki/Privacy).